example/spec/kinesisTests/KinesisTestTriggerSpec.js

```diff
@@ -160,6 +163,7 @@ xdescribe('The Cloud Notification Mechanism Kinesis workflow\n', () => {
 
   beforeAll(async () => {
     // populate collections, providers and test data
+    console.log('\n');
```

Comment:
I pulled it off of the describe(), because it was running on with one of the before all logs. I'll find it and see if it can be added to the other log.